.github/workflows/main-e2e.yml

```diff
@@ -70,16 +70,9 @@ jobs:
           name: pre-req-results
           path: testing/mochawesome-report/report.html
 
-      - name: Rocket.Chat Notification
-        if: always() && github.ref == 'refs/heads/main'
-        uses: ./.github/actions/rocket-chat-notification
-        with:
-          text: '*Pre-Reqs*'
-          job-status: ${{ job.status }}
-          webhook-url: ${{ secrets.ROCKETCHAT_WEBHOOK }}
-
   smoke-test:
     runs-on: ubuntu-latest
+    needs: [pre-reqs]
```

Comment:
Does this make the action wait till the previous step is complete?

Reply:
yes that's right, also requires it to pass